src/ORM/FieldType/DBDate.php

```diff
@@ -271,18 +309,23 @@ public function Rfc822()
     public function Rfc2822()
     {
         if ($this->value) {
-            return date('Y-m-d H:i:s', strtotime($this->value));
+            return date('Y-m-d H:i:s', $this->getTimestamp());
```

Comment:
Shouldn't we use `getFormatter()` consistently here, rather than relying on `date()` for some of the features? It'll be confusing to refer back to two different symbol systems (`date()` and ICU)